### DISCUSSION TITLE
What ethical data analytics tool should we use?

### DISCUSSION THREAD

**Discussion by ecurrencyhodler** [saving-satoshi/saving-satoshi#246] [General]
Let's explore different types of ethical data analytic tools we can use for the project. Some requirements I discussed with Jonas are:

1. Doesn't get funneled to google
2. Doesn't track IP addresses
3. Have the ability to show the data to anyone, including users of the platform

One analytics platform I've used before is Matomo but I'm not sure if it qualifies the 3rd criteria. 

What other requirements should we consider? What kind of data do we want to collect? What tools do people recommend?

**Comment by afterburn** [comment]
What kind of data are we looking to collect? 

Edit: Sorry, I misread your last sentence, I personally think some innocent user statistics would be nice to have, from ChatGPT
```
1. Daily Active Users (DAU): This measures the number of unique users who use a product or service on a daily basis.
2. Monthly Active Users (MAU): This measures the number of unique users who use a product or service on a monthly basis.
3. Time spent on site/app: This measures the amount of time users spend on a website or app, which can help determine how engaged users are with the product.
4. User retention rate: This measures the percentage of users who continue to use a product or service over a specific period of time, such as a month or a year.
5. Churn rate: This measures the percentage of users who stop using a product or service over a specific period of time, such as a month or a year.
6. Conversion rate: This measures the percentage of users who take a desired action, such as making a purchase or signing up for a newsletter, after interacting with a product or service.
7. Click-through rate (CTR): This measures the percentage of users who click on a link or ad after seeing it.
8. Bounce rate: This measures the percentage of users who leave a website after viewing only one page.
9. Net Promoter Score (NPS): This measures how likely users are to recommend a product or service to others.
```

Though I think we should be prioritizing an error logging solution before we think about analytics. To name a few: [Sentry](https://sentry.io/welcome/), [Raygun](https://raygun.com/), [Datadog](https://www.datadoghq.com/), [Bugsnag](https://www.bugsnag.com/). These tools can help us identify issues that we haven't encountered ourselves during QA.

**Reply by ecurrencyhodler** [reply]
Other data points that could be helpful is user completion rate and drop-off rate. This will help us see what challenges may be a point of friction for some. 

I think completion rate and user retention rate can be measured through the info we keep in the backend as we plan to store completion progress and profiles there. But that would require the ability to query our backend.

**Comment by ecurrencyhodler** [comment]
I started doing some research on tools we could use. Looks like we will have to pay or self-host. Feel free to add to the list if you want. I've been using [this site](https://github.com/newTendermint/awesome-analytics#privacy-focused-analytics) for reference.
https://docs.google.com/spreadsheets/d/1oEjgH8A51YC4gRs3rge9ARneO1wU3fHOHe8wEOTjRWM/edit?usp=sharing

**Comment by adamjonas** [comment]
We went with unmami!
